src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java

```diff
@@ -119,8 +119,11 @@ public void verifyThatSurveillanceDataIsSavedInReport(final String friendlySurvI
         /* Workaround to resolve Stale Element Reference Exception-
          * reload the element after form update so it's attached to DOM: https://www.seleniumhq.org/exceptions/stale_element_reference.jsp*/
         getDriver().get(getUrl() + "#/surveillance/reporting");
+        getDriver().navigate().refresh();
```

Comment:
Why does the page need a refresh immediately after load? I see the comment about the workaround; does this actually work? I don't know why this page, specifically, would be any different on initial load than after a refresh

Reply:
It was throwing stale element so i had to refresh page after i initiate report.

Reply:
But this isn't refreshing after initiating the report; it's refreshing after loading the page

Reply:
Cleaned the code